torch/overrides.py

```diff
@@ -183,6 +183,8 @@ def get_ignored_functions() -> Set[Callable]:
         torch.unify_type_list,
         torch.make_dual,
         torch.unpack_dual,
+        torch.is_warn_always,
```

Comment:
I checked similar flags and think we should go with `torch.is_warn_always_enabled` for the public getter name. Sorry about the name thrashing. That's the last name change, I promise.

Reply:
done